digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -797,4 +799,50 @@ protected void setCapacityProviderStrategy(final EcsClientConfig clientConfig, f
             request.setCapacityProviderStrategy(Arrays.asList(capacityProviderStrategyItem));
         }
     }
+
+    private String dumpTaskRequest(final RunTaskRequest request)
```

Comment:
why not static method?

Reply:
Not particular reasons. I will convert this method to static.

Reply:
75ccbff